### PR TITLE
minor tweaks to AWS README

### PR DESCRIPTION
**Pull request by JacobGabrielson**
Issue #, if available:

Description of changes: just a few minor doc updates, upgrade to using 1.19

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.

### CODE REVIEW

File: docs/aws/README.md [hunk 1]
```diff
@@ -25,17 +25,17 @@ eksctl create cluster \
 ```
 
 ### Create IAM Resources
-This command will create IAM resources used by Karpenter. We recommend using Cloud Formation and IAM Roles for Service Accounts to manage these permissions. For production use, please review and restrict these permissions for your use case.
+This command will create IAM resources used by Karpenter. We recommend using [CloudFormation](https://aws.amazon.com/cloudformation/) and [IAM Roles for Service Accounts](https://docs.aws.amazon.com/eks/latest/userguide/iam-roles-for-service-accounts.html) (IRSA) to manage these permissions. For production use, please review and restrict these permissions for your use case.
 ```bash
 aws cloudformation deploy \
-  --stack-name Karpenter \
+  --stack-name Karpenter-${CLUSTER_NAME} \
```
Comment (ellistarn) [inline docs/aws/README.md:31]: I'm a touch cautious of this. What about this config varies by cluster?

Comment (JacobGabrielson) [reply docs/aws/README.md:31]: I wanted to have two stacks but also just wanted to paste this in. Line 34 seems to be cluster-specific (the OpenID...)

**Review by ellistarn — APPROVED** [review]
sgtm

**Comment by JacobGabrielson** [comment]
a few minor doc updates, upgrade to using 1.19

**Pull request merged**